src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -56,6 +56,12 @@ def __init__(self, profile, default_url, logger_name = __name__):
             self.default_headers["Authorization"] = f"Bearer {self.session.tokenValue}"
         elif self.session.type == session_constants.AUTH_TYPE_TOKEN:
             self.default_headers["Cookie"] = f"{self.session.tokenType}={self.session.tokenValue}"
+    
+    def __enter__(self):
+        return self
+
+    def __exit__(self):
```

Comment:
The `__exit__` method needs to take 3 additional parameters (see [PEP 343](https://peps.python.org/pep-0343/) for more details):
```suggestion
    def __exit__(self, exc_type, exception, traceback):
```

Reply:
Yes you are right forgot to add that :sweat_smile: